tests/integration/devtools/test_props_check.py

```diff
@@ -15,22 +15,22 @@
         "fail": True,
         "name": 'missing required "value" inside options',
         "component": dcc.Checklist,
-        "props": {"options": [{"label": "hello"}], "values": ["test"]},
+        "props": {"options": [{"label": "hello"}], "value": ["test"]},
```

Comment:
This is what infuriates me about Percy... OK, here it's just a test problem, not a real bug, but the images were completely different from what they should have been, and got auto-approved, I assume, because the change came from DCC and so likely happened first in master.